Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```
@@ -407,6 +407,23 @@
       - ReagentId: SpaceCleaner
         Quantity: 30
 
+- type: entity
+  parent: SmokeGrenade
+  id: CleanerGrenadeNuke
+  name: the clean solution
+  description: A special grenade made by a specially angry janitor, be aware it will flood the room, and maybe more, with it's cleaning foam.
```

```suggestion
  description: A special cleanade designed by an especially tenacious janitor. Its foam can spread over an entire room, leaving it squeaky clean and smelling slightly of lemons.
```
Original sentence didn't read quite right, due to improper punctuation and a misspelling of the word "especially". I offer a potential replacement, but it's up to you wheterh or not you accept it.